packages/app/src/components-styled/line-chart/line-chart.tsx

```diff
@@ -352,36 +373,27 @@ function formatDefaultTooltip<T extends Value>(
   isPercentage?: boolean
 ) {
   // default tooltip assumes one line is rendered:
-  const value = values[0];
-  const isDaily = isDailyValue(values);
-  const isWeekly = isWeeklyValue(values);
 
-  if (isDaily) {
+  if (isDateSeries(values)) {
+    const value = values[0];
     return (
       <>
         <Text as="span" fontWeight="bold">
-          {formatDateFromMilliseconds(value.__date.getTime()) + ': '}
+          {`${formatDateFromMilliseconds(value.__date.getTime())}: `}
         </Text>
         {isPercentage
           ? `${formatPercentage(value.__value)}%`
           : formatNumber(value.__value)}
       </>
     );
-  } else if (isWeekly) {
+  } else if (isDateSpanSeries(values)) {
+    const value = values[0];
     return (
       <>
         <Text as="span" fontWeight="bold">
-          {formatDateFromSeconds(
-            ((value as unknown) as WeeklyValue).date_start_unix,
-            'short'
-          )}{' '}
-          -{' '}
-          {formatDateFromSeconds(
-            ((value as unknown) as WeeklyValue).date_end_unix,
-            'short'
-          )}
-          :
-        </Text>{' '}
+          {`${formatDateFromSeconds(value.date_start_unix, 'short')} -
```

Comment:
Not sure formatting goes well here, you might want to render it as follows just to be safe:
```suggestion
          {`${formatDateFromSeconds(value.date_start_unix, 'short')}{' - '}
```
(assuming there should be a whitespace _after_ the dash)

Reply:
I prefer formatting things in a single string. I don't quite see a point in breaking things up. To me a single string is more readable.

Reply:
oh wait, I didn't see there were nested string interpolations. In that case I guess rendering will be fine, but the syntax is IMHO not very readable..

Reply:
I think it would help if I first formatted the date strings.... I'll try that